content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -286,74 +335,6 @@ Custom user actions are user actions declared and sent manually via the [`addUse
 {{% /tab %}}
 {{< /tabs >}}
 
-## Default attributes
```

Comment:
Line 261/310 [addUserAction API] should link to real_user_monitoring/browser/tracking_user_actions/?tab=npm#custom-user-actions